ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml

```diff
@@ -80,20 +80,15 @@ operator:
     with: hyperconverged-cluster-webhook
   - pullspec: +ARTIFACTS_SERVER_IMAGE_TO_REPLACE+
     with: virt-artifacts-server
-promotion:
-  additional_images:
-    hyperconverged-cluster-operator-tests: src
-  name: "4.10"
-  namespace: origin
```

Comment:
why is promotion being removed here?

Reply:
upstream we are not really consuming `quay.io/openshift/origin-hyperconverged-cluster-operator:4.11.0` image but  `quay.io/kubevirt/hyperconverged-cluster-operator:1.7.0-unstable` (that will become `1.7.0`) once properly tagged.
quay.io/kubevirt/ images are handled with a "promotion" mechanism on https://prow.ci.kubevirt.io/

Reply:
ok, so you are not reliant on these CI images for anything.  Not even your CI tests need them?  (not just the mirror to quay.io, but the promotion to registry.ci.openshift.org also?  e.g. registry.ci.openshift.org/origin/4.11:hyperconverged-cluster-functest and/or registry.ci.openshift.org/ocp/4.11:hyperconverged-cluster-functest)

Reply:
we have a few CI lanes here and others on `prow.ci.kubevirt.io` and we have a post submit job there that after a merge push our images to quay.io/kubevirt/hyperconverged-cluster-operator:1.7.0-unstable

during our test we are consuming our operator as something like:
          `image: registry.build02.ci.openshift.org/ci-op-fj24ikhz/pipeline@sha256:bd4f8710fdc6b734be5298f12a8213e9db84daee7c13407a192a8a0524425623`
          
 and functest as:
 ```
 + FUNC_TEST_IMAGE=registry.build04.ci.openshift.org/ci-op-6zd369gq/pipeline@sha256:4988c9f988acefe4bcb571d7321c78c201a95fcb539495e760f81266d8a81461
+ echo 'Running tests with registry.build04.ci.openshift.org/ci-op-6zd369gq/pipeline@sha256:4988c9f988acefe4bcb571d7321c78c201a95fcb539495e760f81266d8a81461'
```
    
 so I don't think we need neither ther promotion to `registry.ci.openshift.org`

Reply:
ok, yeah as long as your CI job only consumes images that are built by the repo the PR is associated w/, your fine.  It's if your CI job is consuming images from other repos that you'd need them promoted (or need to consume them from prow.ci.kubevirt.io).

sounds like you're in good shape, so i have no objection to dropping the promotion, and by implication, the mirroring.